webservices/common/models/reports.py

```diff
@@ -333,6 +337,8 @@ class BaseFiling(PdfMixin, CsvFecMixin, db.Model):
     election_state = db.Column('el_state', db.String)
     receipt_date = db.Column('create_dt', db.Date, index=True)
     sign_date = db.Column(db.Date)
+    means_filed = 'e-file'
```

Comment:
Seeing this again, I think we should define an `E_FILE_XXX` constant (variable, I know) and set it to `e-file`, then use that where all of these attributes are defined.

Related, is there some other way this should be defined in the model, such as a `hybrid_property` (e.g., [like this one in our itemized base model](https://github.com/18F/openFEC/blob/develop/webservices/common/models/itemized.py#L24))?